internal/drivers/anka/pool.go

```diff
@@ -0,0 +1,237 @@
+package anka
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/drone-runners/drone-runner-aws/internal/lehelper"
+	"github.com/drone-runners/drone-runner-aws/types"
+	"github.com/drone/runner-go/logger"
+
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	ankabin = "/usr/local/bin/anka"
+)
+
+type ankaShow struct {
+	UUID    string `json:"uuid"`
+	Name    string `json:"name"`
+	Created string `json:"created"`
+	Memory  string `json:"memory"`
+	IP      string `json:"ip"`
+}
+
+func (p *provider) RootDir() string {
+	return p.rootDir
+}
+
+func (p *provider) ProviderName() string {
+	return string(types.ProviderAnka)
+}
+
+func (p *provider) Ping(_ context.Context) error {
+	return nil
+}
+
+func (p *provider) CanHibernate() bool {
+	return false
+}
+
+func (p *provider) Create(ctx context.Context, opts *types.InstanceCreateOpts) (instance *types.Instance, err error) {
+	uData := lehelper.GenerateUserdata(p.userData, opts)
+	machineName := fmt.Sprintf(opts.RunnerName+"-"+"-%d", time.Now().Unix())
+
+	logr := logger.FromContext(ctx).
+		WithField("cloud", types.ProviderAnka).
+		WithField("name", machineName).
+		WithField("pool", opts.PoolName)
+
+	var result []byte
+	cmdCloneVM := commandCloneVM(p.vmID, machineName)
+	_, err = cmdCloneVM.CombinedOutput()
+	if err != nil {
+		return nil, err
+	}
+
+	cmdStartVM := commandAnka(machineName, "start")
+	_, err = cmdStartVM.CombinedOutput()
+	if err != nil {
+		return nil, err
+	}
+	var ip string
+	for i := 1; i <= 60; i++ {
```

Comment:
60 seems like a magic number

Reply:
There's no point in making this configurable - usually only takes about 2-3 secs to get an IP then exits

Reply:
can you put in a comment to explaining what the number is for